test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -1524,6 +1498,106 @@ static public void testCheckCastRefClassOnNull() throws Throwable {
 		checkCastRefClassOnNull.invoke();
 	}
 
+
+	// The three following tests can be used to verify that flattened value types in arrays are handled properly by the GC. In the
```

Comment:
Add -Xint to the playlist.xml java command. There is no JIT support for any of the  VT bytecodes right now.